classes/class-dintero-checkout-templates.php

```diff
@@ -202,7 +208,7 @@ public function iframe() {
 	public function add_body_class( $class ) {
 		if ( is_checkout() && ! is_wc_endpoint_url( 'order-received' ) ) {
 
-			// Don't display Dintero body classes if we have a cart that doesn't need payment.
+			// Dintero is not available for free orders except for free trial subscriptions. Refer to the Subscription class.
```

Comment:
Again, specify what we are referring to in the class to make it easier to lookup in the future.